src/OMSimulatorPython/CMakeLists.txt

```diff
@@ -3,19 +3,27 @@ project(OMSimulatorPython)
 set(OMSIMULATORLIB_DIR_STRING ${HOST_SHORT})
 IF (WIN32 AND MSVC)
   set(OMSIMULATORLIB_STRING "OMSimulator.dll")
+  set(OMSIMULATOR_TLM_STRING "omtlmsimulator.dll")
 ELSEIF (WIN32 AND MINGW)
   set(OMSIMULATORLIB_STRING "libOMSimulator.dll")
+  set(OMSIMULATOR_TLM_STRING "libomtlmsimulator.dll")
 ELSE ()
   set(OMSIMULATORLIB_STRING "libOMSimulator.so")
+  set(OMSIMULATOR_TLM_STRING "libomtlmsimulator.so")
 ENDIF ()
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/OMSimulator.py.in" "${CMAKE_CURRENT_BINARY_DIR}/OMSimulator.py" @ONLY)
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OMSimulator.py" DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
 
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OMSimulator.py" DESTINATION ${CMAKE_INSTALL_PREFIX}/OMSimulator)
+install(FILES "${CMAKE_INSTALL_PREFIX}/bin/${OMSIMULATORLIB_STRING}" DESTINATION ${CMAKE_INSTALL_PREFIX}/OMSimulator)
+install(FILES "${CMAKE_INSTALL_PREFIX}/bin/${OMSIMULATOR_TLM_STRING}" DESTINATION ${CMAKE_INSTALL_PREFIX}/OMSimulator)
+
 IF (WIN32 AND MSVC)
   install(FILES OMSimulatorPython3.bat DESTINATION ${CMAKE_INSTALL_PREFIX}/bin PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
 ELSEIF (WIN32 AND MINGW)
   install(FILES OMSimulatorPython3.bat DESTINATION ${CMAKE_INSTALL_PREFIX}/bin PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
   install(FILES OMSimulatorPython3 DESTINATION ${CMAKE_INSTALL_PREFIX}/bin PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
+  install(FILES "${CMAKE_INSTALL_PREFIX}/bin/libomtlmsimulator.dll" DESTINATION ${CMAKE_INSTALL_PREFIX}/OMSimulator)
```

Comment:
This should be removed, right?

Reply:
removed https://github.com/OpenModelica/OMSimulator/pull/795/commits/5c162140aa719d488a73e5b698295444fc64e648